### ISSUE TITLE
Mode at startup is not correct

### ISSUE THREAD

**Issue by FuriouZz** [dj95/zjstatus#55]
**Describe the bug**
The current mode at startup is not correct.

**To Reproduce**
1. Set `default_mode "locked"` in your configuration
2. Run `zellij` with your configuration
3. Check the mode in the statusbar

**Expected behavior**
The mode in the status bar must be LCK

**Screenshots**
![CleanShot 2024-05-06 at 17 57 28](https://github.com/dj95/zjstatus/assets/1800468/b51c0dff-0578-4d31-96d4-9bf015178969)

**Desktop (please complete the following information):**
 - OS: Mac Sonoma 14.4.1
 - Zellij version v0.40.0
 - Version v0.14.0

**Layout**

How does the layout look like? Please copy it into a code block.

```kdl
layout {
    tab name="editor" focus=true hide_floating_panes=true {
        pane name="editor" borderless=true {
            command "hx"
        }
    }

    default_tab_template {
        children
        floating_panes {
            pane { 
                // plugin location="zjpane"
            }
        }
        pane size=1 borderless=true {
            plugin location="zjstatus"
        }
        pane size=1 borderless=true {
            plugin location="status-bar"
        }
    }
}
```

```kdl
// config.kdl
plugins {
	zjstatus location="file:~/.config/zellij/plugins/zjstatus.wasm" {
        format_left  "{mode}{tabs}"
        format_right "#[fg=#b8bb26,bold]{swap_layout} #[fg=#83a598,bold]{command_git_branch}"

        hide_frame_for_single_pane "true"

        mode_normal  "#[fg=#292C33,bg=blue,bold] NOR "
        mode_locked  "#[fg=#292C33,bg=green,bold] LCK "
        mode_resize  "#[fg=#292C33,bg=red,bold] RSZ "
        mode_pane    "#[fg=#292C33,bg=magenta,bold] PNE "
        mode_tab     "#[fg=#292C33,bg=magenta,bold] TAB "
        mode_scroll  "#[fg=#292C33,bg=yellow,bold] SCR "
        mode_session "#[fg=#292C33,bg=yellow,bold] SSN "
        mode_move    "#[fg=#292C33,bg=red,bold] MOV "
        mode_tmux    "#[fg=#292C33,bg=green,bold] TMX "

        tab_normal "#[fg=blue] {name}"
        tab_active "#[fg=blue,bold] {name}*"

        command_git_branch_command    "git rev-parse --abbrev-ref HEAD"
        command_git_branch_format     "#[fg=blue] {stdout} "
        command_git_branch_interval   "10"
        command_git_branch_rendermode "static"

        datetime          " {format} "
        datetime_format   "%H:%M %d-%b-%y"
        datetime_timezone "Europe/Berlin"
    }
}

default_mode "locked"
```

**Additional context**

**Comment by FuriouZz** [comment]
Nice! It seems to work well on 0.14.1-rc1

**Comment by dj95** [comment]
Thanks for the confirmation. I released 0.14.1 right now, as the impact of the bug seems to be quite annoying.
